packages/sdk/src/4.0/methodsInterface.ts

```diff
@@ -3663,23 +3664,21 @@ export interface ILooker40SDK extends IAPIMethods {
    * Any UDD (a dashboard which exists in the Looker database rather than as a LookML file) which has a `lookml_link_id`
    * property value referring to a LookML dashboard's id (model::dashboardname) will be updated so that it matches the current state of the LookML dashboard.
    *
+   * If the dashboard_ids parameter is specified, only the dashboards with the specified ids will be updated.
+   *
    * For this operation to succeed the user must have permission to view the LookML dashboard, and only linked dashboards
    * that the user has permission to update will be synced.
    *
    * To **link** or **unlink** a UDD set the `lookml_link_id` property with [update_dashboard()](#!/Dashboard/update_dashboard)
    *
    * PATCH /dashboards/{lookml_dashboard_id}/sync -> number[]
    *
-   * @param lookml_dashboard_id Id of LookML dashboard, in the form 'model::dashboardname'
-   * @param body Partial<IWriteDashboard>
-   * @param raw_locale If true, and this dashboard is localized, export it with the raw keys, not localized.
+   * @param request composed interface "Partial<IRequestSyncLookmlDashboard>" for complex method parameters
    * @param options one-time API call overrides
    *
    */
   sync_lookml_dashboard(
-    lookml_dashboard_id: string,
-    body: Partial<IWriteDashboard>,
-    raw_locale?: boolean,
+    request: Partial<IRequestSyncLookmlDashboard>,
```

Comment:
we should have a note in the changelog that this method signature changed

Reply:
Ack